### ISSUE TITLE
Compatibility Hint

### ISSUE THREAD

**Issue by sonkkkay** [Berry-Pool/nami-wallet#210]
I've read a number of threads in here already but I am still confused. All my ADA, NFT and other CNT are stored in my Ledger Nano which I have been using it with Yoroi, so there is no way I can just simply Connect my hardware wallet to Nami wallet and have access/see all my assets without having to restore my seed phrase into Nami or manually send everything out to Nami (this 2nd option sounds quite costly)? Thanks

**Comment by alessandrokonrad** [comment]
Yes you can connect a HW to Nami. If you really just want to use a HW, you still have to import a seed phrase wallet first, even if you don't use it. When you are in the wallet then you can connect a HW and also import different accounts from it.

**Comment by sonkkkay** [comment]
Thanks for getting back to me. I was able to connect the hw wallet but the problem is that it wont show every CNT and NFT that I have in my hw wallet because the hw wallet seed phrase was used to recover in another wallet (yes, I recovered my hw once). It suggests that I either send all my assets from my hw to Nami wallet or recover my hw seed phrase on Nami which I dont really like either options.
